package/src/contexts/messagesContext/MessagesContext.tsx

```diff
@@ -7,7 +7,7 @@ import type { TouchableOpacityProps } from 'react-native';
 import type { ChannelState, MessageResponse } from 'stream-chat';
 
 import type { Alignment } from '../messageContext/MessageContext';
-import type { MessageAction } from '../messageOverlayContext/MessageOverlayContext';
+import type { MessageActionListItemMainProps } from '../../components/MessageOverlay/MessageActionListItem';
 import type { SuggestionCommand } from '../suggestionsContext/SuggestionsContext';
 import type { DeepPartial } from '../themeContext/ThemeContext';
 import type { Theme } from '../themeContext/utils/theme';
```

Comment:
```suggestion
import type { DeepPartial } from '../themeContext/ThemeContext';
import type { MessageActionListItemMainProps } from '../../components/MessageOverlay/MessageActionListItem';
import type { SuggestionCommand } from '../suggestionsContext/SuggestionsContext';
import type { Theme } from '../themeContext/utils/theme';
```

Reply:
I didn't understand what do we need to do here. Can you explain?

Reply:
It's just sorting the imports alphabetically